manifests/kustomize/env/platform-agnostic/mysql/mysql-deployment.yaml

```diff
@@ -15,11 +15,15 @@ spec:
       labels:
         app: mysql
     spec:
+      serviceAccountName: mysql
       containers:
-      - env:
+      - args:
+        - --datadir
+        - /var/lib/mysql/datadir
+        env:
         - name: MYSQL_ALLOW_EMPTY_PASSWORD
           value: "true"
-        image: gcr.io/ml-pipeline/mysql:5.6
+        image: docker.io/mysql:5.7
```

Comment:
due to recent stricter rate limit of dockerhub, I'd suggest one of the following:

* keep using the old image
* ask me to port `docker.io/mysql:5.7` on `gcr.io/ml-pipeline/mysql:5.7`

which do you prefer?

Reply:
With mysql:8.0

```
[julius@localhost ~]$ kubectl -n kubeflow logs ml-pipeline-7c56db5db9-ts547 -c ml-pipeline-api-server 
I0310 18:49:55.490470       8 client_manager.go:134] Initializing client manager
I0310 18:49:55.490564       8 config.go:50] Config DBConfig.ExtraParams not specified, skipping

(Error 1046: No database selected) 
[2021-03-10 18:49:55]  

(Error 1046: No database selected) 
[2021-03-10 18:49:55]  

(Error 1046: No database selected) 
[2021-03-10 18:49:55]  

(Error 1046: No database selected) 
[2021-03-10 18:49:55]  

(Error 1046: No database selected) 
[2021-03-10 18:49:55]  

(Error 1046: No database selected) 
[2021-03-10 18:49:55]  

(Error 1046: No database selected) 
[2021-03-10 18:49:55]  
F0310 18:49:55.518981       8 client_manager.go:212] Failed to initialize the databases.
```

so i guess we should stay with 5.7 for the moment. It will be interesting to see how the metadata stuff will work with 5.7 instead of 8.0